src/webconnector-tellus/src/scraper-cursor.js

```diff
@@ -64,11 +64,15 @@ export default function(table, scraperMapping, token, doneCallback) {
 
   function slurpCursorAPI() {
     let page = 1;
-
+    const pageSize = limit > 0 && limit < defaultParams.page_size ? limit : defaultParams.page_size;
```

Comment:
Actual fetched `page_size` isn't changed. So this only yields invalid progress messages... No real need to do a `min` operation for `limit` and `defaultParams.page_size`. Just get the default page_size.